lib/ansible/modules/apt.py

```diff
@@ -30,11 +30,13 @@
     description:
       - Indicates the desired package state. C(latest) ensures that the latest version is installed. C(build-dep) ensures the package build dependencies
         are installed. C(fixed) attempt to correct a system with broken dependencies in place.
+    type: str
     default: present
     choices: [ absent, build-dep, latest, present, fixed ]
   update_cache:
     description:
       - Run the equivalent of C(apt-get update) before the operation. Can be run as part of the package installation or as a separate step.
+    aliases: ['update-cache']
```

Comment:
```suggestion
    aliases: [ update-cache ]
```

To match other parameter styles.